static/locales/en-US.yaml

```diff
@@ -226,35 +223,32 @@ Settings:
     Select Import Type: Select Import Type
     Select Export Type: Select Export Type
     Import Subscriptions: Import Subscriptions
-    Import FreeTube: Import FreeTube
-    Import YouTube: Import YouTube
-    Import NewPipe: Import NewPipe
+    Import FreeTube: FreeTube Import
+    Import YouTube: YouTube Import
+    Import NewPipe: NewPipe Import
     Check for Legacy Subscriptions: Check for Legacy Subscriptions
     Export Subscriptions: Export Subscriptions
     Export FreeTube: Export FreeTube
     Export YouTube: Export YouTube
-    Export NewPipe: Export NewPipe
+    Export NewPipe: NewPipe Export
     Import History: Import History
     Export History: Export History
     Profile object has insufficient data, skipping item: Profile object has insufficient
-      data, skipping item
-    All subscriptions and profiles have been successfully imported: All subscriptions
-      and profiles have been successfully imported
+  data, skipping item
+    All subscriptions and profiles have been successfully imported: Imported all
+  subscriptions and profiles
```

Comment:
I'd still add the 'successfully' at the end

Reply:
There is (presumably) no import that can be unsuccessful, and success is to be expected(?).

Reply:
I mean it can fail if something in the data is wrong.

Reply:
Yes, but in the event that it would, it would say so(?).
There is no reason to question the default, and that is how it is said.
"Successfully" isn't added unless there is reason to.
It is just not good, and worse yet when translated directly.